package.json

```diff
@@ -94,7 +74,6 @@
     "preact-compat": "^3.11.0",
     "promise-rat-race": "^1.4.0",
     "proptypes": "^0.14.3",
-    "semver": "^5.3.0",
```

Comment:
I have a feeling this is needed in production too, in order for the app to decide which version of n-ui css to pull from the network